nix-buffer.el

```diff
@@ -152,42 +157,56 @@ EVENT The process status change event string."
 	  (kill-buffer out-buf)
 	  (kill-buffer err-buf))))))
 
-(defun nix-buffer--nix-build (root expr-file)
+(defun nix-buffer--nix-build (expr-file &optional root skip-safety)
```

Comment:
Erm, why is `root` optional?

Reply:
I guess it will break anything that needs 'root' like haskellMonoRepo. My thinking was in the context of 'nix-buffer-with-string' you wouldn't want to have a 'root' path set. For instance, you could use it in a script that didn't have any directory or buffer associated.